doc/sphinx/source/esmvaldiag/observations.inc

```diff
@@ -1,7 +1,269 @@
-.. _observations:
-
 ************************************************************
 Contributing a CMORizing script for an observational dataset
 ************************************************************
 
+ESMValTool is designed to work with CF compliant data and 
```

Comment:
Maybe it would be good make CF compliant a link to the CF conventions

Reply:
Done.